_tutorials-recipes/understanding-dynamic-global-properties.md

```diff
@@ -126,7 +124,13 @@ Total asset held in confidential balances.
 
 ### `total_vesting_fund_steem`<a style="float: right" href="#sections"><i class="fas fa-chevron-up fa-sm" /></a>
 
-STEEM that is invested in STEEM POWER (powered up).
+<ul style="float: right; list-style: none;">
```

Comment:
Nice on following the existing standard for deprecated notices.

Still, as I look this is not good practice. Visually or semantically.

I'll add a commit that addresses both those issues.